1/Dockerfile

```diff
@@ -48,6 +48,35 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod -R 777 /etc/alternatives && \
+    chmod -R 777 /var/lib/alternatives && \
+    chmod -R 777 /usr/lib/jvm && \
+    chmod 777 /usr/bin && \
+    chmod 777 /usr/lib/jvm-exports && \
+    chmod 777 /usr/share/man/man1 && \
```

Comment:
chgrp'ing to 0 should also work (as long as the dir has group rwx permissions). 

I don't know that it's actually any more secure at the end of the day, though it feels slightly more secure.